catt/cli.py

```diff
@@ -160,6 +160,7 @@ def cli(ctx, device):
     help="YouTube-DL option. "
     "Should be passed as `-y option=value`, and can be specified multiple times (implies --force-default).",
 )
+@click.option("--seek", type=CATT_TIME, metavar="TIME", help="Start playback at specific timestamp.")
```

Comment:
Could we rename this to "seek-to"? I think that's a bit more explicit as a name, what do you think?

Reply:
You're right. Fixed.